### ISSUE TITLE
Do we support tf.nn.bidirectional_dynamic_rnn ?

### ISSUE THREAD

**Issue by mullerhai** [tensorflow/java#421]
Hi:
      I only see CudnnRNN ,the project has not testcase for it,I not know if we support  tf.nn.bidirectional_dynamic_rnn or normal simple rnn?
     thank you

**Comment by karllessard** [comment]
I don't think we do support out-of-the-box RNN ops, most in TensorFlow has been also deprecated in favor to Keras RNN cells and layers, so I guess this support will also as we port the Keras API into our framework.

**Comment by mullerhai** [comment]
> I don't think we do support out-of-the-box RNN ops, most in TensorFlow has been also deprecated in favor to Keras RNN cells and layers, so I guess this support will also as we port the Keras API into our framework.

wow,  when this framework will wrapper keras api?

**Comment by karllessard** [comment]
This is a own project on its own, leaded by @JimClarke5 , a lot of work as already been done by porting the optimizers, metrics, losses, regularizers and activations of Keras [to Java](https://github.com/tensorflow/java/tree/master/tensorflow-framework/src/main/java/org/tensorflow/framework). The plumbing for layers and models are coming on soon and from there it would be interesting to tackle the support for RNNs.

If you are interested to know more about it @mullerhai and if you can, I suggest that you join the next [SIG meeting](https://docs.google.com/document/d/1RqSIKKFLE8kMSjUuZWsVS4JNCRHIrCYiuWcFZRNk2oU/edit?usp=sharing) where we discuss various topics related to TF Java.
